### PR TITLE
Update field to handle only list for values and defaults

### PR DESCRIPTION
**Pull request by GreatWizard**
refs #172 

review: @xcambar

### CODE REVIEW

**Comment by GreatWizard** [comment]
ping @xcambar

**Comment by GreatWizard** [comment]
ping @xcambar

File: app/components/builder-field-settings.js [hunk 1]
```diff
@@ -221,6 +222,18 @@ export default Component.extend({
       this.set('buffer.required', false);
     }
   }),
+  didBufferValueChange: observer('buffer.value', function() {
+    let bufferValue = this.get('buffer.value');
+    if (bufferValue) {
+      this.set('buffer.defaults', makeArray(bufferValue));
```
Comment (xcambar) [inline app/components/builder-field-settings.js]: s/defaults/values/ ?

Comment (xcambar) [inline app/components/builder-field-settings.js]: You could use a computed getter/setter

**Comment by GreatWizard** [comment]
Update with computed instead of observer

**Comment by GreatWizard** [comment]
ping

**Comment by xcambar** [comment]
👍

**Pull request merged**